projects/storefrontlib/src/cms-components/asm/asm/asm.component.ts

```diff
@@ -1,18 +1,32 @@
 import { Component, OnInit } from '@angular/core';
-import { AuthService, UserToken } from '@spartacus/core';
-import { Observable } from 'rxjs';
+import { AuthService, User, UserService, UserToken } from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { switchMap, take } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-asm',
   templateUrl: './asm.component.html',
 })
 export class AsmComponent implements OnInit {
   csAgentToken$: Observable<UserToken>;
+  customer$: Observable<User>;
 
-  constructor(private auth: AuthService) {}
+  constructor(
+    protected auth: AuthService,
+    protected userService: UserService
+  ) {}
 
   ngOnInit(): void {
     this.csAgentToken$ = this.auth.getCustomerSupportAgentToken();
+    this.customer$ = this.customer$ = this.auth.getUserToken().pipe(
```

Comment:
Why is it `this.customer$ = this.customer$ =` and not `this.customer$ =`?

Reply:
That's a mistake.

Fixed.